src/components/DataDictionary.tsx

```diff
@@ -249,18 +242,13 @@ export const DataDictionary: React.FC<{}> = () => {
           </PageContent>
         </PageLayout>
       </div>
-    </ThemeProvider>
+    </ComponentsProvider>
   )
 }
 
 const PageHeader = styled(Flex as any)`
-  background-color: ${theme.colors.key};
-  background-position: 100% 0;
-  background-repeat: no-repeat;
-  background-size: 836px 120px;
+  background-color: ${theme.colors.inform};
```

Comment:
I'd leave this as `key` and specify themeCustomizations in the ComponentsProvider (suggestion above).

Reply:
Tried your suggestion but impact was to large. Hard coding to '#1A73E8' with a big fat todo. 

Hopefully Jeremy will drive a theming overhaul